ts/screens/PreferencesScreen.tsx

```diff
@@ -0,0 +1,57 @@
+import { Body, Container, Content, H1, Text, View } from "native-base";
+import * as React from "react";
+import PreferenceList from "../components/PreferencesList";
+import ScreenHeader from "../components/ScreenHeader";
+import AppHeader from "../components/ui/AppHeader";
+import I18n from "../i18n";
+import { PreferenceItem } from "../types/PreferenceItem";
+
+const preferences: ReadonlyArray<PreferenceItem> = [
+  {
+    id: "email",
+    icon: require("../../img/wallet/icon-avviso-pagopa.png"),
```

Comment:
icon needs to be specific for the item, not pagopa in this case

Reply:
That icon is just a placeholder.

Reply:
then leave a note for the reviewer :)

Reply:
I had already done it alongside the comment with the screenshots.